frontend/scripts/actions/tool.actions.js

```diff
@@ -82,6 +76,59 @@ const _reloadLinks = (param, value, type, reloadLinks = true) => dispatch => {
   }
 };
 
+const _setRecolorByAction = (recolorBy, state) => {
```

Comment:
Do we have some convention to use prefix `_` for private functions? I think as long is not exported it's clearly private.

Reply:
we do, but we don't enforce it that much